plugins/meta/tuning/tuning.go

```diff
@@ -165,6 +183,122 @@ func changeMtu(ifName string, mtu int) error {
 	return netlink.LinkSetMTU(link, mtu)
 }
 
+func createBackup(ifName, containerID, backupPath string, tuned bool, tuningConf *TuningConf) error {
+	config := configToRestore{Tuned: tuned}
+	if tuned {
+		link, err := netlink.LinkByName(ifName)
+		if err != nil {
+			return fmt.Errorf("failed to get %q: %v", ifName, err)
+		}
+		if tuningConf.Mac != "" {
+			config.Mac = link.Attrs().HardwareAddr.String()
+		}
+		if tuningConf.Promisc {
+			config.Promisc = new(bool)
+			*config.Promisc = (link.Attrs().Promisc != 0)
+		}
+		if tuningConf.Mtu != 0 {
+			config.Mtu = link.Attrs().MTU
+		}
+	}
+
+	backupDir := path.Join(backupPath, containerID)
```

Comment:
Rather than creating a directory per container, just join container id + ifname - that's what we do in other plugins.

Reply:
Done :)

Reply:
Oh, I forgot about tests. Fixed.